go-controller/pkg/ovn/namespace.go

```diff
@@ -90,8 +91,30 @@ func (oc *Controller) addPodToNamespace(ns string, ips []*net.IPNet) (*gatewayIn
 	}
 
 	return oc.getRoutingExternalGWs(nsInfo), oc.getRoutingPodGWs(nsInfo), nil
+}*/
+
+// TEMP
+// addPodToNamespace adds the pod's IP to the namespace's address set and returns
+// pod's routing gateway info
+func (oc *Controller) addPodToNamespaceReturnsOps(ns string, ips []*net.IPNet) (*gatewayInfo, map[string]*gatewayInfo, []ovsdb.Operation, error) {
```

Comment:
@tssurya I would just make `addPodToNamespace()` always return the ops instead of having a separate function, since I think there's only one place its called?

Reply:
you are right, its only called from `addLogicalPort`, I wanted to run this in airflow to see if numbers got better before refactoring it., but I'll eventually what you are suggesting.